tests/discovery/test_linkedin_links.py

```diff
@@ -12,5 +13,16 @@ def test_get_links(self):
         links = search.get_links()
         assert list(links)
 
+    def test_links_linkedin(self):
+        f = open("test_linkedin_links.txt").read()
```

Comment:
Can you update single char variable names please say what they are doing to make it better for correctiveness and code understanding